m2cgen/assemblers/fallback_expressions.py

```diff
@@ -66,6 +64,118 @@ def log1p(expr):
         utils.div(utils.mul(expr, ast.LogExpr(expr1p)), expr1pm1))
 
 
+def atan(expr):
```

Comment:
This looks super scary :D

Reply:
Yeah, I know! 😃 

But it was the only one I could find with appropriate accuracy. Other either have error about 1e-4 - 1e-6 or look even more scary. I even cannot remember how many trials I've made. There were Chebyshev, CORDIC, Taylor, Remez (in [Sollya package](http://sollya.gforge.inria.fr/sollya-2.9/help.php?name=remez))...

Reply:
Wow, impressive. Amazing work as always 👍

Reply:
Thank you!